raiden/tests/integration/test_blockchainservice.py

```diff
@@ -75,7 +75,12 @@ def test_new_netting_contract(raiden_network, asset_amount, settle_timeout):
     assert manager0.channels_by_participant(peer0_address) == [netting_address_01]
     assert manager0.channels_by_participant(peer1_address) == [netting_address_01]
     assert manager0.channels_by_participant(peer2_address) == []
-
+    with pytest.raises(Exception):
```

Comment:
This tests passes, but that is only because you are not testing if it works when trying to open a channel with the same partner after settling. With your current implementation this would not throw an exception after settling. I would like to have a test that tries to open a new channel between two parties that have already settled.